src/blockchain/sync-augur-node-with-blockchain.ts

```diff
@@ -12,12 +12,16 @@ export function syncAugurNodeWithBlockchain(db: Knex,  augur: Augur, ethereumNod
   augur.connect({ ethereumNode: ethereumNodeEndpoints }, (): void => {
     console.log("Waiting for first block...");
     startAugurListeners(db, augur, (blockNumber: string): void => {
-      db.raw(`SELECT "highestBlockNumber" FROM blockchain_sync_history ORDER BY "highestBlockNumber" DESC LIMIT 1`)
-        .asCallback( (err: Error|null, rows?: Array<HighestBlockNumberRow>): void => {
+      db.max("block as highestBlockNumber").from(function(this: Knex.QueryBuilder): void {
+        this.max("highestBlockNumber as block").from("blockchain_sync_history").unionAll(function(this: Knex.QueryBuilder): void {
+          this.max("blockNumber as block").from("blocks");
+        }).as("maxBlocks");
+      }).asCallback( (err: Error|null, rows?: Array<HighestBlockNumberRow>): void => {
           if (err) return callback(err);
+          console.log(rows);
           const row: HighestBlockNumberRow = rows![0];
           const fromBlock: number = (!row || !row.highestBlockNumber) ? uploadBlockNumbers[augur.rpc.getNetworkID()] : row.highestBlockNumber + 1;
-          const highestBlockNumber: number = parseInt(blockNumber, 16);
+          const highestBlockNumber: number = parseInt(blockNumber, 16) - 1;
```

Comment:
why `-1` ?

Reply:
We receive block number N off the chain, and then sync the blocks before that which were unsynched.

```
Getting Augur logs from block 1377804 to block 1903664
new block: 1903665 1508440891
new block: 1903666 1508440893
```